wqu_app/message.py

```diff
@@ -30,17 +34,10 @@ def get_weather(coords):
 
 
 def greet(ip_address):
-    coords, city, region = get_geolocation(ip_address)
+    #coords, city, region = get_geolocation(ip_address)
+    coords, city = get_geolocation(ip_address)
```

Comment:
This is a nit pick but should we use a different variable name instead of `city`? Perhaps `city_region` or something else? What are your thoughts?